libs/libcommon/src/libcommon/queue.py

```diff
@@ -449,6 +504,8 @@ def create_jobs(self, job_infos: List[JobInfo]) -> int:
                 )
                 for job_info in job_infos
             ]
+            for job in jobs:
+                increase_metric(queue=job.type, status=Status.WAITING)
```

Comment:
would it make sense to do only one query with an array of job_types? Or is it unnecessary optimization?

Reply:
Not sure if it is more efficient to perform one __in at db level for update instead of a loop.
Also if would anyway have to keep the order-combination of job_type-status for the update (cannot do job_type__in and status__in).
Maybe with a bulk write but not sure if it is worth to add too much logic to replace a loop. wdyt?